CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 x.x.x Release notes (yyyy-MM-dd)
 =============================================================
+NOTE: The synchronization protocol version has been bumped from 29 to 30. The server side remains compatible with protocol versions back to, and including 22. The client side remains compatible with protocol versions back to, and including 26 (Realm JavaScript v2.27.0).
```

Comment:
It's not clear to me what the user will gain with this info?
We have not communicated protocol versions in the past. There is no change in compatibility either.
So I would remove this.

What could be relevant to note perhaps is that we provide a better error message (that needs to be implemented in OS likely)? So this is not enough to gain anything from the upgrade.

Reply:
We did mention protocol update in v2.27.0.

True, in order to get a better error message, we need to do something in object store.

Reply:
But there is also very little risk. To the extent that there is significant risk in Sync 4.9.0, it is all on the server side.

I see no point in holding it back.

cc @geragray

Reply:
That's a mistake in 2.27.0. We shouldn't really do that as it's an implementation detail to us. What we need to tell users is which SDK versions are compatible with which ROS versions. Including an indirection (protocol version) is not helpful. We talked about this before. So we should only update the Docs that reflects this and is referenced in the release notes.

Reply:
I'll remove it from the change log.